scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/cql/Cell.java

```diff
@@ -14,5 +14,5 @@
 
     boolean isDeleted();
     
-    ByteBuffer getUnsafeBytes();
+    ByteBuffer getAsUnsafeBytes();
```

Comment:
How about adding a docstring comment explaining what this function does? It would make it even more obvious what this function does when somebody looks at the interface. Besides, I think that modern Java IDEs usually display docstring comments along with suggestions (maybe I'm wrong, I don't have much experience with Java), so this message should be helpful for users writing code using this library.

Reply:
In the amended commit, I added a docstring for this method. You are right about great IDE support for those.

Originally, I was planning to add Javadocs to all exposed classes/methods "in one go" (which didn't happen yet...), but now I see the value of adding those "one-by-one".